### PR TITLE
Add Rescale! netkan

### PR DESCRIPTION
**Pull request by Galileo88**

### CODE REVIEW

**Comment by Galileo88** [comment]
i will add "conflicts" as soon as these are merged.

File: NetKAN/Rescale-10.netkan [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_10.zip",
```
Comment (Olympic1) [inline NetKAN/Rescale-10.netkan]: - `download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_10.zip"`

File: NetKAN/Rescale-10625.netkan [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_10.625.zip",
```
Comment (Olympic1) [inline NetKAN/Rescale-10625.netkan]: `download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_10.625.zip"`

File: NetKAN/Rescale-25.netkan [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_2.5.zip",
```
Comment (Olympic1) [inline NetKAN/Rescale-25.netkan]: `download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_2.5.zip"`

File: NetKAN/Rescale-32.netkan [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_3.2.zip",
```
Comment (Olympic1) [inline NetKAN/Rescale-32.netkan]: `download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_3.2.zip"`

File: NetKAN/Rescale-64.netkan [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_6.4.zip",
```
Comment (Olympic1) [inline NetKAN/Rescale-64.netkan]: `download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_6.4.zip"`

**Comment by Galileo88** [comment]
@Olympic1 all changes made

**Comment by Olympic1** [comment]
Ok, thanks. Merging

**Pull request merged**